internal/core/ports/claims_service.go

```diff
@@ -207,6 +207,7 @@ type GetCredentialQrCodeResponse struct {
 // ClaimsService is the interface implemented by the claim service
 type ClaimsService interface {
 	Save(ctx context.Context, claimReq *CreateClaimRequest) (*domain.Claim, error)
+	GetRevoked(ctx context.Context, currentState string) ([]*domain.Claim, error)
```

Comment:
```suggestion
	GetRevoked(ctx context.Context, currentState string) ([]domain.Claim, error)
```
I find it more clear if we return []domain.Claim instead of slice of pointers, as this list should never contain nil values

Reply:
I'm closing the comment as this change will affect a lot of places that should not be affected by this PR.